applications/audio/broadcast_config_tool/src/main.c

```diff
@@ -1004,6 +1119,511 @@ static int cmd_location(const struct shell *shell, size_t argc, char **argv)
+	char *encrypt1_argv[4] = {"encrypt", "1", "1", "Auratest"};
+
+	char *context1_argv[4] = {"context", "media", "1", "0"};
+
+	char *program_info1_argv[4] = {"program_info", "Biathlon", "1", "0"};
+
+	char *imm1_argv[4] = {"immediate", "1", "1", "0"};
+
+	char *num_bis1_argv[4] = {"num_bises", "2", "1", "0"};
+
+	char *locationFL1_argv[5] = {"location", "FL", "1", "0", "0"};
+	char *locationFR1_argv[5] = {"location", "FR", "1", "0", "1"};
+
+	cmd_preset(shell, 3, preset1_argv);
+	cmd_adv_name(shell, 3, adv_name1_argv);
+	cmd_name(shell, 3, name1_argv);
+	cmd_packing(shell, 3, packing1_argv);
+	cmd_encrypt(shell, 4, encrypt1_argv);
+
+	cmd_context(shell, 4, context1_argv);
+
+	cmd_program_info(shell, 4, program_info1_argv);
+
+	cmd_immediate_set(shell, 4, imm1_argv);
+
+	cmd_num_bises(shell, 4, num_bis1_argv);
+
+	cmd_location(shell, 5, locationFL1_argv);
+	cmd_location(shell, 5, locationFR1_argv);
+
+	cmd_show(shell, 0, NULL);
+}
+
+static void multilanguage_set(const struct shell *shell)
+{
+	char *num_subs_argv[3] = {"num_subgroups", "3", "0"};
+	char *preset_argv[3] = {"preset", "24_2_2", "0"};
+	char *adv_name_argv[3] = {"adv_name", "Multi-language", "0"};
+	char *name_argv[3] = {"name", "Multi-language", "0"};
+	char *packing_argv[3] = {"packing", "int", "0"};
+
+	char *lang0_argv[4] = {"lang_set", "eng", "0", "0"};
+	char *lang1_argv[4] = {"lang_set", "spa", "0", "1"};
+	char *lang2_argv[4] = {"lang_set", "nor", "0", "2"};
+
+	char *context0_argv[4] = {"context", "unspecified", "0", "0"};
+	char *context1_argv[4] = {"context", "unspecified", "0", "1"};
+	char *context2_argv[4] = {"context", "unspecified", "0", "2"};
+
+	char *num_bis0_argv[4] = {"num_bises", "1", "0", "0"};
+	char *num_bis1_argv[4] = {"num_bises", "1", "0", "1"};
+	char *num_bis2_argv[4] = {"num_bises", "1", "0", "2"};
+
+	cmd_num_subgroups(shell, 3, num_subs_argv);
+	cmd_preset(shell, 3, preset_argv);
+	cmd_adv_name(shell, 3, adv_name_argv);
+	cmd_name(shell, 3, name_argv);
+	cmd_packing(shell, 3, packing_argv);
+
+	cmd_lang_set(shell, 4, lang0_argv);
+	cmd_lang_set(shell, 4, lang1_argv);
+	cmd_lang_set(shell, 4, lang2_argv);
+
+	cmd_context(shell, 4, context0_argv);
+	cmd_context(shell, 4, context1_argv);
+	cmd_context(shell, 4, context2_argv);
+
+	cmd_num_bises(shell, 4, num_bis0_argv);
+	cmd_num_bises(shell, 4, num_bis1_argv);
+	cmd_num_bises(shell, 4, num_bis2_argv);
+
+	cmd_show(shell, 0, NULL);
+}
+
+static void personalsharing_set(const struct shell *shell)
+{
+	char *preset_argv[3] = {"preset", "48_2_2", "0"};
+	char *adv_name_argv[3] = {"adv_name", "John's phone", "0"};
+	char *name_argv[3] = {"name", "Personal sharing", "0"};
+	char *packing_argv[3] = {"packing", "int", "0"};
+	char *encrypt_argv[4] = {"encrypt", "1", "0", "Auratest"};
+
+	char *context_argv[4] = {"context", "conversational", "0", "0"};
+
+	char *num_bis_argv[4] = {"num_bises", "2", "0", "0"};
+
+	char *locationFL_argv[5] = {"location", "FL", "0", "0", "0"};
```

Comment:
Not very important, but maybe we could add "_" before FL/FR.

Reply:
Not sure I understand. You want the argument to be _FL? Meaning the command for setting front left for bis 0 would be `bct location _FL 0 0`?

Reply:
I was thought the name of variable could be "location_FL_argv", but I'm not sure that is better or not now.

Reply:
Oh, that makes more sense :P Yeah, I can change that